examples/utils/jabberwocky.xml

```diff
@@ -1,8 +1,25 @@
 <ul>
-    <li>a</li>
+    <li>
+        a
+        <ul>
+            <li>a.a</li>
+            <li><p>a.b</p></li>
+            <li><h1>a.c</h1></li>
+            <li>a.d</li>
+            <li>
+                <ul>
+                    <li>a.d.a</li>
+                </ul>
+            </li>
+        </ul>
+    </li>
     <li>b</li>
+    <li>
+        <ol>
+            <li>b.1</li>
```

Comment:
Backspace at the beginning of this line doesn't work: we get
```html
<paragraph>
    b<paragraph>b.1</paragraph> ...
</paragraph>
```
which is then rendered as
```html
<li>
    b<p>b.1</p> ...
</li>
```